tests/e2e/e2e_test.go

```diff
@@ -61,14 +69,14 @@ func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
 	address2 := node.CreateWalletAndFund("addr2", fundTokens)
 	address3 := node.CreateWalletAndFund("addr3", fundTokens)
 
-	// Create 2 positions for node1: overlap together, overlap with 2 node3 positions)
+	// Create 2 positions for address1: overlap together, overlap with 2 address3 positions, balanced
```

Comment:
What does it mean to be balanced?

Reply:
sorry, it is not related to this pr, will remove it